lms/templates/dashboard/_dashboard_credit_info.html

```diff
@@ -1,75 +1,87 @@
 <%page args="credit_status" />
 <%!
-    import datetime
-    import pytz
     from django.utils.translation import ugettext as _
-    from util.date_utils import get_default_time_display
 %>
 <%namespace name='static' file='../static_content.html'/>
-% if credit_status["provider_name"]:
-    <% provider_link='<a href="{}" target="_blank">{}</a>'.format(credit_status["provider_status_url"], credit_status["provider_name"]) %>
-% endif
+
 
 % if credit_status["eligible"]:
+    <%
+        provider_link = '<a href="{href}" target="_blank">{name}</a>'.format(
+            href=credit_status["provider_status_url"],
+            name=credit_status["provider_name"])
+
+        error = credit_status['error']
+
+
+        status = 'eligible'
+
+        # Translators: provider_name is the name of a credit provider or university (e.g. State University)
+        credit_msg = _("You have completed this course and are eligible to purchase course credit. Select <strong>Get Credit</strong> to get started.")
+        credit_msg_class = "credit-eligibility-msg"
+        credit_btn_class = "purchase-credit-btn"
+        credit_btn_label = _("Get Credit")
+        credit_btn_href = '{root}/credit/checkout/{course_id}/'.format(
+            root=settings.ECOMMERCE_PUBLIC_URL_ROOT,
+            course_id=credit_status['course_key'])
+
+        if credit_status["purchased"]:
+            request_status = credit_status["request_status"]
+            if request_status is None:
+                # Learner must initiate the credit request
+
+                # Translators: link_to_provider_site is a link to an external webpage. The text of the link will be the name of a  credit provider, such as 'State University' or 'Happy Fun Company'.
+                credit_msg = _("Thank you for your payment. To receive course credit, you must now request credit "
+                "at the {link_to_provider_site} website. Select <b>Request Credit</b> to get started.").format(
+                    link_to_provider_site=provider_link,
+                )
+                credit_msg_class = "credit-request-pending-msg"
```

Comment:
Yes. This is "correct". This class is only uses for tests. I will update it to actually be correct in a follow-up PR.